src/pages/PullRequestPage/subroute/CommitsTab/CommitsTab.jsx

```diff
@@ -11,9 +15,16 @@ const Loader = () => (
 )
 
 function CommitsTab() {
+  const { provider, owner } = useParams()
```

Comment:
I've really come to like these sort of "intermediate" files, where their sole purpose is to direct you to a different "version" of what you want to see. I suppose we didn't have the need for this before, but I find this so readable and easy to follow, pat on our backs

Reply:
One other nice thing about this, is that with this pattern we're only loading the JS for the path we're taking thanks Suspense!